thrift.js

```diff
@@ -305,4 +407,26 @@ Thrift.prototype.resolveMapConst = function resolveMapConst(def) {
     return map;
 };
 
+Thrift.prototype.resolveIdentifier = function resolveIdentifier(def, identifier) {
```

Comment:
resolveIdentifierValue (to be distinguished from resolveIdentifier which would return a type)